src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ConstructedEETypeNode.cs

```diff
@@ -137,11 +137,11 @@ protected override DependencyList ComputeNonRelocationBasedDependencies(NodeFact
 
             if (closestDefType.HasGenericDictionarySlot())
```

Comment:
If we're going to play the blame game, I added the original `if` block with different content in https://github.com/dotnet/corert/commit/e05cca67d48cf8d4afb167245271157ebc3755eb. Then people put other stuff in the block. I eventually removed the one line that was in the `if` block originally, leaving just the stuff other people added :P.